massa-execution-exports/src/test_exports/mock.rs

```diff
@@ -106,6 +106,14 @@ impl ExecutionController for MockExecutionController {
         response_rx.recv().unwrap()
     }
 
+    fn get_final_and_active_data_entry(
+        &self,
+        _addr: &Address,
+        _key: &massa_hash::Hash,
```

Comment:
To avoid differences in the code I think we should import massa_hash::Hash at the top of the file like the other files.

Reply:
Done.